packages/fxa-content-server/app/tests/spec/views/app.js

```diff
@@ -342,15 +410,79 @@ describe('views/app', function () {
       before(function () {
         createDeps();
 
-        sinon.spy(view, 'fatalError');
+        sinon.spy(notifier, 'trigger');
+        sinon.spy(view, 'setTitle');
 
-        return view.showView(ViewThatErrors, {});
+        return view
+          .showView(ViewThatRenders, {
+            viewName: 'some-other-view',
+            surveyTargeter: getSurveyTargeter()
+          });
       });
 
-      it('writes the error to the DOM', function () {
-        assert.isTrue(view.fatalError.calledWith(renderError));
+      it('does not the survey to the DOM', function () {
+        assert.equal($('.survey-wrapped').length, 0);
       });
     });
+
+    describe('with a survey, but no targeter', function () {
+      var ViewThatRenders = Backbone.View.extend({
+        afterVisible: sinon.spy(),
+        destroy: sinon.spy(),
+        logView: sinon.spy(),
+        render() {
+          this.$el.html('<div id="rendered-view"></div>');
+          return Promise.resolve(true);
+        },
+        titleFromView() {
+          return 'the title';
+        },
+      });
+
+      before(function () {
+        createDeps();
+
+        sinon.spy(notifier, 'trigger');
+        sinon.spy(view, 'setTitle');
+
+        return view
+          .showView(ViewThatRenders, {
+            viewName: 'some-other-view'
+          });
+      });
+    });
+
+    it('does not the survey to the DOM', function () {
+      assert.equal($('.survey-wrapped').length, 0);
+    });
+  });
+
+  describe('with a view that errors', function () {
+    var renderError = AuthErrors.toError('UNEXPECTED_ERROR');
+
+    var ViewThatErrors = Backbone.View.extend({
+      afterVisible: sinon.spy(),
+      destroy: sinon.spy(),
+      logView: sinon.spy(),
+      render: sinon.spy(function () {
+        return Promise.reject(renderError);
+      }),
+      titleFromView() {
+        return 'the title';
+      },
+    });
+
+    before(function () {
+      createDeps();
+
+      sinon.spy(view, 'fatalError');
+
+      return view.showView(ViewThatErrors, {});
+    });
+
+    it('writes the error to the DOM', function () {
+      assert.isTrue(view.fatalError.calledWith(renderError));
+    });
```

Comment:
Is this an effect of the diff or is this a new test?  It does not appear to be related to surveys.

Reply:
This appears to be a diff issue. It's not a duplicate test so I don't want to remove it. Not sure why its showing up in the diff